rules-reviewed/eap7/eap71/tests/hibernate51-53.windup.test.xml

```diff
@@ -0,0 +1,190 @@
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-383-
+            <rule id="hibernate51-53-00700-test">
+                <when>
+                    <not>
+                        <iterable-filter size="5">
+                            <hint-exists message="" />
+                        </iterable-filter>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-384
+            <rule id="hibernate51-53-00800-test">
+                <when>
+                    <not>
+                        <iterable-filter size="1">
+                            <hint-exists message="" />
+                        </iterable-filter>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-385
+            <rule id="hibernate51-53-00900-test">
+                <when>
+                    <not>
+                        <iterable-filter size="5">
+                            <hint-exists message="" />
+                        </iterable-filter>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-386
+            <rule id="hibernate51-53-01000-test">
+                <when>
+                    <not>
+                        <iterable-filter size="1">
+                            <hint-exists message="" />
+                        </iterable-filter>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-387
+            <rule id="hibernate51-53-01100-test">
+                <when>
+                    <not>
+                        <iterable-filter size="5">
+                            <hint-exists message="" />
+                        </iterable-filter>
+                    </not>
+                </when>
+                <perform>
+                    <fail message="" />
+                </perform>
+            </rule>
+            -->
+            <!--https://issues.jboss.org/browse/WINDUPRULE-388-->
+            <rule id="hibernate51-53-01200-test">
+                <when>
+                    <not>
+                        <hint-exists message="Any references to org.hibernate.*.getType().*" />
```

Comment:
Fixed the test to work with the new hint's message and added the iteration to check that there are exactly 2 issues found
```suggestion
                        <iterable-filter size="2">
                            <hint-exists message="Any references to `org.hibernate.*.getType().*" />
                        </iterable-filter>
```